app.json

```diff
@@ -1213,7 +1287,8 @@
         "learnmode": {
           "instruction": {
             "en": "Toggle the bulb to start pairing.\n\nIf pairing does not automatically start within 20 seconds, try resetting the device by toggling it 6 times with a 5 second interval. It will flash to indicate it has been reset.",
-            "nl": "Zet de lamp uit en weer aan.\n\nWordt de lamp niet automatisch toegevoegd binnen 20 seconden, probeer dan de lamp te resetten door hem 6 keer aan en uit te zetten met 5 seconden tussenpozen. De lamp zal knipperen om aan te geven dat hij is gereset."
+            "nl": "Zet de lamp uit en weer aan.\n\nWordt de lamp niet automatisch toegevoegd binnen 20 seconden, probeer dan de lamp te resetten door hem 6 keer aan en uit te zetten met 5 seconden tussenpozen. De lamp zal knipperen om aan te geven dat hij is gereset.",
+            "de": "Schalten Sie die Lampe aus und wieder an um das Pairing zu starten.\n\nWenn das Pairing nicht automatisch innerhalb von 20 Sekunden beginnt, versuchen Sie die Lampe zurückzusetzen, indem Sie sie mit einem 5 Sekunden Intervall 6 Mal ein- und ausschalten. Zur Bestätigung blinkt die Lampe."
```

Comment:
The text might identical to line 858 (LED Strip), right?
If so, please change the text.
"de": "Schalten Sie den Strip aus und wieder an um das Pairing zu starten.\n\nWenn das Pairing nicht automatisch innerhalb von 20 Sekunden beginnt, versuchen Sie den Strip zurückzusetzen, indem Sie ihn 6 Mal sehr schnell ein- und ausschalten. Zur Bestätigung blinkt der Strip zweimal."